### PR TITLE
DOCSP-32494 Add Query Converter Error Include

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Add note where the query converter errors are located.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-32494/code-generation/query-converter/convert-views/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-32494/code-generation/query-converter/convert-triggers/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-32494/code-generation/query-converter/convert-stored-procedures/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-32494/code-generation/query-converter/convert-queries/

## JIRA

https://jira.mongodb.org/browse/DOCSP-32494

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6629562fb67dbf804aeefaef

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/fact-query-converter-failure-errors.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+If the query converter has any errors, you can view the details under
```
Comment (sarah-olson-mongodb) [inline source/includes/fact-query-converter-failure-errors.rst:1]: [minor nit]
You might omit "any" here, in the name of fewer words.

**Review by sarah-olson-mongodb — APPROVED** [review]
LGTM @ianf-mongodb, with one optional and tiny suggestion.

And very nice use of includes. Thanks!

**Comment by ianf-mongodb** [comment]
@tomhollander this is ready for a looky.

**Review by tomhollander — APPROVED** [review]

File: source/includes/fact-query-converter-failure-errors.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+If the query converter has errors, you can view the details under
```
Comment (mdb-ashley) [inline source/includes/fact-query-converter-failure-errors.rst:1]: [suggestion] We have two prepositions here. Did you want _under_ or _in_?

Comment (ianf-mongodb) [reply source/includes/fact-query-converter-failure-errors.rst]: in

**Pull request merged**